pkg/vdri/trustbloc/vdri.go

```diff
@@ -130,14 +131,25 @@ func (v *VDRI) Read(did string, opts ...vdriapi.ResolveOpts) (*docdid.Doc, error
 
 	var doc *docdid.Doc
 
+	var docBytes []byte
+
 	for _, e := range endpoints {
 		resp, err := v.sidetreeResolve(e.URL, did, opts...)
 		if err != nil {
 			return nil, err
 		}
 
-		// TODO add logic to compare response from each endpoint
+		respBytes, err := resp.JSONBytes()
```

Comment:
i think an improvement would be to use JSON Canocilization Scheme (JCS). We added JCS for LD signatures.

Reply:
see https://github.com/trustbloc/json-gold/tree/0.3.0-update/ld/internal/jsoncanonicalizer